binders/kafka-binder/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaBinderConfiguration.java

```diff
@@ -78,11 +77,12 @@
  * @author Oleg Zhurakousky
  * @author Artem Bilan
  * @author Aldo Sinanaj
+ * @author Chris Bono
  */
 @Configuration(proxyBeanMethods = false)
 @ConditionalOnMissingBean(Binder.class)
-@Import({ KafkaAutoConfiguration.class, KafkaBinderHealthIndicatorConfiguration.class })
```

Comment:
I think this was done especially for multi-binder scenario, when we even have several Kafka binder instances in the same stream application.
So, every binder got its own auto-configured beans based exactly on their specific properties.
Therefore how it is going to work from now on?

Thanks

Reply:
@artembilan , yep this is for the multi-binder case and described [here](https://github.com/spring-cloud/spring-cloud-stream/issues/2349#issuecomment-1108893702). However, the only beans that we actually use are KafkaProperties, none of the other beans that are brought in by KAC are actually used (AFAICT).

Reply:
I believe @garyrussell comment [here](https://github.com/spring-cloud/spring-cloud-stream-binder-rabbit/issues/187#issuecomment-440777907) also points to only using the properties for the binder.

Reply:
Sure! I see now. After closer look indeed Kafka binder doesn't use an auto-configured `KafkaTemplate` and friends like `ProducerFactory` or `ConsumerFactory`.
So, yeah... It might rally work for Kafka binder, but that is not the case for Kinesis. It does rely on the auto-configured `AWSCredentialsProvider` and `RegionProvider`. Sorry for misleading.
We may revise that in favor of "manual" auto-configuration against respective properties like it is done for Kafka binder.

The Rabbit binder does rely on the `ConnectionFactory rabbitConnectionFactory`, though. See `RabbitMessageChannelBinderConfiguration`.
That might be revised respectively.

I guess different issue to stop polluting here 😄 

Thank you!

Reply:
OK. I think I have missed a couple rounds. I see now there is this one `RabbitConfiguration`. And looks like the `RabbitBinderConfiguration` does not import any auto-configurations any more.
Sorry for a noise.

Reply:
> Sorry for a noise.

Nws. It is not noise to me and, well, it used to import _RabbitAutoConfiguration_ but I added [this to SB](https://github.com/spring-projects/spring-boot/pull/26982) a while back to break the auto-config dependence. 

> but that is not the case for Kinesis. It does rely on the auto-configured AWSCredentialsProvider and RegionProvider. Sorry for misleading.
We may revise that in favor of "manual" auto-configuration against respective properties like it is done for Kafka binder.

Ahh good to know. Do you mind creating a ticket to break dependency on KAC for the Kinesis binder and I will follow on w/ that work once the dust settles on this proposal?